postgresql/src/main/scala/org/scalarelational/postgresql/PostgreSQLDatastore.scala

```diff
@@ -0,0 +1,145 @@
+package org.scalarelational.postgresql
+
+import javax.sql.DataSource
+
+import org.postgresql.ds.PGSimpleDataSource
+import org.powerscala.log.{Level, Logging}
+import org.powerscala.property.Property
+import org.scalarelational.column.ColumnPropertyContainer
+import org.scalarelational.column.property.{AutoIncrement, Polymorphic, Unique}
+import org.scalarelational.datatype._
+import org.scalarelational.instruction.ddl.CreateColumn
+import org.scalarelational.model._
+import org.scalarelational.op.{Condition, RegexCondition}
+import org.scalarelational.result.ResultSetIterator
+
+import scala.collection.mutable.ListBuffer
+
+/**
+ * @author Robert Djubek <envy1988@gmail.com>
+ */
+
```

Comment:
Create an object ``PostgreSQL`` in ``org/scalarelational/postgresql/`` with the following ``case class``es.